app/views/trips/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<!-- <h1>trips#show</h1>
+<p>Find me in app/views/trips/show.html.erb</p> -->
+
+<% if @trip.nil? %>
+  <h1> 404 Not found</h1>
+<% else %>
+<div class="container">
+  <table>
+    <tr>
+      <th>ID</th>
+      <th>Date</th>
+      <th>Rating</th>
+      <th>Cost</th>
+      <th>Driver</th>
+      <th>Passenger</th>
+      <th>Edit</th>
+      <th>Delete</th>
+    </tr>
+    <tr>
+      <td><%= link_to "#{@trip.id}", trip_path(@trip.id) %></td>
+      <td><%="#{@trip.date}" %></td>
+      <td><%="#{@trip.rating}" %></td>
+      <td><%="$#{@trip.convert_money(@trip.cost)}" %></td>
+      <td><%=link_to "#{@trip.driver_id}", driver_path(@trip.driver_id) %></td>
```

Comment:
I'd recommend showing the passenger and driver's names instead of ids.

Reply:
Hi Chris, could you or Devin go over how to do this with me? I tried to set @driver and @passenger in the trip show method so I could use it in this view as @driver.name/@passenger.name but was not successful.